auth/keycloak/config/hs_err_pid385.log

```diff
@@ -0,0 +1,1325 @@
+#
```

Comment:
I doubt we want this in source.

Reply:
Oooops!!, will remove it. And yes the zoom level will be saved in redux